modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -452,22 +452,17 @@
 							if (typeof callback === 'function') {
 								callback(imageSrc, selectedItem);
 							} else {
-								var elementOuterHtml =
```

Comment:
We need to keep `elementOuterHtml` so that the image gets printed out in the CKEditor.